Content.Server/Atmos/Piping/Trinary/EntitySystems/GasFilterSystem.cs

```diff
@@ -211,5 +213,18 @@ private void OnFilterAnalyzed(EntityUid uid, GasFilterComponent component, GasAn
 
             args.DeviceFlipped = inlet != null && filterNode != null && inlet.CurrentPipeDirection.ToDirection() == filterNode.CurrentPipeDirection.ToDirection().GetClockwise90Degrees();
         }
+
+        private void OnMapInit(EntityUid uid, GasFilterComponent filter, ref MapInitEvent args) // Frontier - Enable on map init
+        {
+            if (!filter.StartEnabled)
+                return;
+
+            filter.Enabled = true;
+
```

Comment:
this is really picky but `GasVolumePumpSystem` reads exactly the same but doesn't have the newline between `Enabled` and `UpdateAppearance`